### DISCUSSION TITLE
Kitty window closes after vagrant up or docker up -d and I'd like it to stay open with zsh running

### DISCUSSION THREAD

**Discussion by kfrohwein** [kovidgoyal/kitty#3218] [Q&A] [answered]
Currently I have to following session script:
```new_tab akona
title akona
layout grid

# akona vagrant
cd ~/Projects/akona
title akona
launch vagrant up       
launch zsh 

# akona cms
cd /home/kfrw/Projects/akona-cms
title akona-cms
launch docker-compose up -d
launch zsh
```

So I open 4 Windows and the vagrant and docker window will close after booting up. This is okay but I would like to `vagrant ssh` into the VM after it booted.

I tried `launch vagrant up && zsh` or `launch zsh -c 'vagrant up'` but regardless what I do it will close the window and I don't know how to prevent that.

And what I really want is `vagrant up && vagrant ssh` which runs fine in the terminal if I create a tab by hand. I then end up in the shell of the virtual maschine.

**Comment by kovidgoyal** [comment, marked as answer]
launch zsh -c "whatever; read"

**Reply by kfrohwein** [reply]
Thanks a lot!
